docs/manual/src/Overview.md

```diff
@@ -1,6 +1,7 @@
 # UniFFI
 
-UniFFI is a tool that automatically generates foreign-language bindings targeting Rust libraries.  
+UniFFI is a tool that automatically generates foreign-language bindings targeting Rust libraries.
+It can be found on [github](https://github.com/mozilla/uniffi-rs/)
```

Comment:
super nit: Maybe a comma after this statement or have it run with the previous line since In the markdown preview it reads "It can be found on github It fits in..." vs "It can be found on github, It fits..." conversely could say "The repo can be found on github".

Reply:
oops, nice catch, thanks.